openvasd/openvasd.c

```diff
@@ -112,7 +112,7 @@ enum openvas_request_method
 
 typedef enum openvas_request_method openvasd_req_method_t;
 
-/** @brief Initialize a notus info struct and stores the server URL
+/** @brief Initialize a openvasd info struct and stores the server URL
  *
  *  @param server Original server to store and to get the info from
  *
```

Comment:
The return doc should be adjusted, it always returns the struct (or aborts on out of mem but we normally just imply that).